Separates Traitor EMP Flashlights from the EMP Kit, reduces TC prices

PKPenguin321 opened the pull request:
This PR removes EMP flashlights from the traitor EMP kit and readds them as a unique and separate item, and changes the price of kits and the flashlight to 2 TCs.

In basically any argument against silicons/cameras, EMPs are brought up as a hard counter, when people fail to realize that it's a whole quarter of your TCs and they're considering it to be basically mandatory. If it's so mandatory, it shouldn't cost so much.

The EMP kit also has too much utility in one purchase to validate a price reduction, so it's functionality has now been spread out to two items.

~~**This PR is entirely experimental and mainly to gather feedback. I don't fully expect it to be merged. Feel free to call me a fucking retard, or, if you're an intelligent person, give actual feedback and thoughts on the issue.**~~ Feedback has mostly been gotten and now I have an idea I'm happy with.

:cl: PKPenguin321
tweak: Traitor EMP kits have had their cost reduced from 5 to 2, and no longer contain the EMP flashlight.
rcsadd: EMP flashlights can now be bought separately for 2 TCs.
/:cl:

PKPenguin321 committed:
reduces price of EMP kit drastically

from 5 to 1

Zilenan91 commented:
Gonna repost my thoughts from the forums 

>Ehhhhhh I think EMPs are at an okay price right now tbh, just cheap enough to do the function you want while allowing room for other stuff. If they were any lower every murderboner and their mother would be buying them to spam EMP's whenever they even began to smell jackboots.

PKPenguin321 commented:
Maybe that could be solved by separating the implant from the kit and putting it into it's own unique purchase?

Incoming5643 commented:
The flashlight alone is worth more than 1TC

Zilenan91 commented:
Nah, that still wouldn't solve the problem, the problem lies with how cameras function. Easier accessability of EMPs would just make sec even more impotent at stopping anything than they are now.

CPTANT commented:
power outage disabling camera's would be a good change.

KorPhaeron commented:
>The flashlight alone is worth more than 1TC

You know you can whack someone with it to disable their headset right? Combined with any sort of stun you have a ghetto parasting

PKPenguin321 commented:
>power outage disabling camera's would be a good change.

rustles silicon player jimmies too much to ever feasibly get merged unfortunately

CPTANT commented:
I just burned down a crew of 50 men as AI and even I support such a change.

Zilenan91 commented:
They can kindly go in the corner because that would be a change that would improve the game drastically.

PKPenguin321 commented:
if silicon players went in the corner ever secborg would be removed by now

Zilenan91 commented:
There's not really a whole lot more to be said

LanCartwright commented:
4 tc, meh.
3 tc sure.
2 tc pushing it.

but one?
really?

PKPenguin321 commented:
okay, how about this:
we separate EMP flashlights from the EMP kit, and make both the flashlights and the kit cost 2 TCs
that way the silent kills tool and the emergency EMP burst tools are separate, and if you only want one method of EMPing, you only have to drop a 10th of your TCs

Zilenan91 commented:
But why would you ever buy anything other than the flashlight, it EMP's shit in an AOE.

Also once again this is still skirting around the higher problem here. If this was merged it would push the game into a state of : 

Do I have a hard counter to X? If so I win. Does X have a hard counter to me? If so I lose.

This would effectively be turning large chunks of the game into Xenomorph levels of hard counters, and the only reason they haven't been fixed up is because they're so rare. I can't imagine how painful every traitor round would be.

PKPenguin321 commented:
>a traitor item enables traitors to kill people
>this is a bad thing

wew

no but seriously, have you considered that that may be part of the goal

and if EMPs are too strong to actually be given to antagonists for any more than a full quarter of their TCs, maybe we should make more things like cyborgs or cyberimplants have more weaknesses than EMPs?

Zilenan91 commented:
It's not that killing people is bad, it's that killing a LOT of people consistently is bad. It makes the game into a non-game, where everyone but the antagonists and security are fodder who may as well suicide if they don't get antag. This is bad. More easily accessed flashlight EMPs would ensure that this non-game comes to be.

LanCartwright commented:
it's not even the EMP to me. it's the implant. Unless you have weapons unaffected by EMP, there's little counterplay to it as a sec officer.

PKPenguin321 committed:
removes the EMP flashlight from the EMP kit

PKPenguin321 committed:
changes emp kit price and desc, adds emp light to uplinks

both cost 2 TCs. emp lights now have a small chance of showing up in surplus crates

PKPenguin321 commented:
PR updated, check the OP

Zilenan91 commented:
I still say it's a bad idea :-1:

PKPenguin321 commented:
Well what would you have me do? We obviously need to give out more EMPs if they're the only counterplay/the only *good* counterplay to half the electronic things on the station.

PKPenguin321 committed:
fixes conflicts in uplink_item.dm

hopefully, anyways
specifically im fixing the conflicts caused by #15286

PKPenguin321 commented on code/modules/uplink/uplink_item.dm:
```diff
@@ -871,6 +878,12 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	item = /obj/item/weapon/c4
 	cost = 1
 
+/datum/uplink_item/device_tools/c4bag
```
shit in my fuck

PKPenguin321 replied:
okay seriously if anybody knows how to fix this hit me up, otherwise im remaking the PR (which sucks because then all this feedback will be harder to find)

Zilenan91 commented:
Handing out more EMPs is the wrong way to do it since all it does is make it easier to silently kill lots of people

The correct way to do it would be to make cameras run off the powernet, but that's very likely out of the scope of this pr.

CPTANT commented:
Making silent kills easier **IS GOOD**

Anyway, I like separating the flash light from the implant and grenades.

PKPenguin321 commented:
welp, sorry folks. as much as i hate to do this i'm gonna remake the PR, i have no clue how to fix the fuckup in the last commit without reintroducing merge conflicts.
